frontend/features/page/sections/HeroSection/HeroSectionBlock.tsx

```diff
@@ -0,0 +1,213 @@
+  "heroSection",
+  {
+    id: string;
+    title: Nullable<string>;
+    subtitle: Nullable<string>;
+    areBubblesActive: boolean;
+  }
+>;
+
+const StyledInlineTextarea = chakra(InlineTextarea);
+
+export function HeroSectionBlock(data: HeroSectionBlockData) {
+  return (
+    <Box
+      overflow="hidden"
+      zIndex="-1"
+      w="full"
+      pb={{
+        base: "80",
+        lg: "400px",
+      }}
+      pt={{
+        base: "52",
+        lg: "80",
+      }}
+      as="section">
+      <Flex
+        flexDir="column"
+        w={{
+          base: "full",
+        }}
+        m={{
+          base: "0 auto",
+        }}>
+        <Flex
+          w={{
+            base: "full",
+            xl: "1200px",
+          }}
+          py="0"
+          px={{
+            base: "6",
+            sm: "10",
+            xl: "0",
+          }}
+          m="0 auto"
+          pos="relative">
+          {data.areBubblesActive === true ? (
+            <Box
+              w="full"
+              pos="absolute"
+              zIndex="-1"
+              left="0"
+              top="0"
+              className="HeroIllustrator">
+              <Bubble
+                top={{
+                  base: "-250px",
+                  sm: "-200px",
+                }}
+                right={{
+                  base: "-200px",
+                  sm: "-200px",
+                }}
+                boxSize="600px"
+                backgroundColor="#f8f1ff"
+              />
+              <Bubble
+                top="200px"
+                right="500px"
+                boxSize="280px"
+                backgroundColor="#ffefe4"
+                display={{
+                  base: "none",
+                  lg: "block",
+                }}
+              />
+              <Bubble
+                top="130px"
+                left="-60px"
+                backgroundColor="#f8f1ff"
+                boxSize="150px"
+                display={{
+                  base: "none",
+                  sm: "block",
+                }}
+              />
+              <Bubble
+                top="-100px"
+                left="400px"
+                backgroundColor="#e8fbf6"
+                boxSize="100px"
+                display={{
+                  base: "none",
+                  lg: "block",
+                }}
+              />
+              <Bubble
+                top="230px"
+                left="230px"
+                backgroundColor="#e8fbf6"
+                boxSize="60px"
+                display={{
+                  base: "none",
+                  xl: "block",
+                }}
+              />
+              <Bubble
+                top="-20px"
+                left="230px"
+                backgroundColor="#fff7df"
+                boxSize="60px"
+                display={{
+                  base: "none",
+                  md: "block",
+                }}
+              />
+            </Box>
+          ) : null}
```

Comment:
When you are using only one branch of the ternary (e.g. `condition ? ... : null`) then you can just use the shortest && form (e.g. `condition && ...`)